test/index.js

```diff
@@ -1,34 +1,38 @@
 "use strict"
-var test = require("tap").test
-var requireInject = require("require-inject")
+const tap = require('tap');
```

Comment:
Semicolons are verboten in this file too. Also I see no value in importing the top level tap object... standard use is to import the test symbol and use that, as it was.